### PR TITLE
Mentions autocomplete popup

### PR DESCRIPTION
**Pull request by MarcoPolo**
@keybase/react-hackers 

This is the mvp and it's feature flagged on only for admins. I think there's still a bit of polishing, but it's probably usable and worth playing around with.

### CODE REVIEW

File: shared/chat/hud/index.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+  >
+    {avatar}
+    <Text type="Body" style={{marginLeft: globalMargins.tiny}}>{username}</Text>
+    <Text type="Body" style={{marginLeft: globalMargins.tiny}}>{fullName}</Text>
+  </ClickableBox>
+)
+
+const Hud = ({style, data, rowRenderer}: Props<*>) => (
+  <Box style={{...hudStyle, ...style}}>
+    <List items={data} renderItem={rowRenderer} />
```
Comment (buoyad) [inline shared/chat/hud/index.js]: Would using ReactList directly here make it easier to access the [scrollAround(i)](https://github.com/orgsync/react-list#scrollaroundindex) fcn to auto-scroll when the user moves into the overflow?

Comment (MarcoPolo) [reply shared/chat/hud/index.js]: List is meant to be a simple cross platform interface. Maybe the better thing would be to add scrollAround functionality to `List` and use `scrollAround` under the hood (and implement something similar for mobile.

But it's also not obvious why you'd want that for this?

Comment (MarcoPolo) [reply shared/chat/hud/index.js]: Ah it's so you can see the visible item if you get to it from kb shortcuts.

Comment (buoyad) [reply shared/chat/hud/index.js]: Also as it is when the list of autocompletes is longer than the hud container (i.e. typing `@m` in keybasefriends) and you arrowDown past the container, you'd still want to see the highlighted profile. When I tested the behavior was that it'd just select past the container into items that aren't visible. But yeah I agree that putting `scrollAround` in List would make sense

**Pull request merged**